components/eventing-controller/controllers/subscription/test_utils.go

```diff
@@ -0,0 +1,229 @@
+package subscription
```

Comment:
Can we choose a different package name ? e.g. subscription_testing ?
These functions are exposed in our public API atm.

Reply:
See godoc screenshot:
<img width="908" alt="Screenshot 2022-03-14 at 4 14 37 PM" src="https://user-images.githubusercontent.com/8013145/158202547-acb1b8ba-41e7-41e9-b7c4-129eb46fbb7d.png">